### DISCUSSION TITLE
How to scale a running cluster when deployed with Operator

### DISCUSSION THREAD

**Discussion by Mopsgeschwindigkeit** [emqx/emqx-operator#762] [Q&A] [answered]
Hello :)
I am running a EMQX5 cluster inside k8s managed by the your operator, and wanted to ask if it's possible to add cores/replicants to an already running cluster?

**Comment by Rory-Z** [comment, marked as answer]
Yes, of course. You can modify the `.spec.coreTemplate | replicantTemplate.spec.replicas` to scale them.
